package.json

```diff
@@ -63,10 +63,14 @@
     "prop-types": "^15.7.2",
     "react": "^16.8.2",
     "react-dom": "^16.8.2",
```

Comment:
please remove this addition from your PR, if you need to add something in the package.json you need a separate PR for that

Reply:
Thank you so much for letting me know.
I have created a separate PR for react icons. Can you have a look?

 https://github.com/HackYourFuture-CPH/fp-class18/pull/63